core/Db/Schema/Mysql.php

```diff
@@ -43,15 +44,27 @@ public function getTablesCreateSql()
             'user'    => "CREATE TABLE {$prefixTables}user (
                           login VARCHAR(100) NOT NULL,
                           password VARCHAR(255) NOT NULL,
-                          alias VARCHAR(45) NOT NULL,
+                          alias VARCHAR(100) NOT NULL,
```

Comment:
Any reason for changing that? If so it should also be changed by the update script

Reply:
Forgot to push that which I did now. Shouldn't have been changed and corrected it. Eventually alias will be removed anyway